src/components/dashboard/SendQRSummary.js

```diff
@@ -39,12 +45,36 @@ const SendQRSummary = ({ screenProps }: AmountProps) => {
     const profile = await userStorage.getUserProfile(to)
     setProfile(profile)
   }
+
+  const checkSurvey = value => {
+    setSurvey(value)
+    setDialogSurvey(true)
+  }
+
   useEffect(() => {
     if (to) {
       updateRecepientProfile()
     }
   }, [to])
 
+  useEffect(() => {
+    if (Config.isEToro && dialogSurvey) {
```

Comment:
this code is used twice, extract all this to surveysend and export as showSurveyDialog